### ISSUE TITLE
[BUG] Pure-mm strategy fails to initialize without external pricing source

### ISSUE THREAD

**Issue by dennisocana** [CoinAlpha/hummingbot#1265]
**Describe the bug**

If I want to use the default market order book mid price instead of external pricing source, the strategy fails to initialize.

This happens in the current development branch. The strategy is working fine in master branch 0.21.0

<img width="674" alt="putty_1suiS7x60M" src="https://user-images.githubusercontent.com/50150287/71651939-87fd7f80-2d5c-11ea-9643-967c225917a4.png">

**Steps To Reproduce**

1. Create a Hummingbot instance in development branch
2. Set up a pure market making strategy by completing the configuration walkthrough
3. Enter `n` when prompted `Would you like to use an external pricing source for mid-market price? (y/n) >>>` then start the strategy

For existing Hummingbot instance:

1. Update to most recent development branch and restart Hummingbot
2. Sending `config` command would ask for config password and will prompt for the newly added parameter `Would you like to use an external pricing source for mid-market price? (y/n) >>>`
3. Enter `n` and start the strategy

**Release version**

```
"branch": "development",
"commit": "7f73abc040c894e6e87fdd887b5bc508bd6a910b",
"date": "2020-01-01T04:58:19Z",
```

**Attachments**

[pure_market_making.log](https://github.com/CoinAlpha/hummingbot/files/4014920/pure_market_making.log)

**Comment by Nullably** [comment]
A PR #1267 should fix this.

**Comment by Hashira-Lars** [comment]
Confirm Fix #1267 tested with the most recent dev image merged with the pr 

                "branch": "development",
                "commit": "73fd3a475cd9c54457560be8924c35cde2396083",
                "date": "2020-01-02T22:45:21Z",
